snmp/datadog_checks/snmp/parsing/metrics.py

```diff
@@ -417,7 +417,16 @@ def _parse_table_metric_tag(mib, parsed_table, metric_tag):
 
 def _parse_column_metric_tag(mib, parsed_table, metric_tag):
     # type: (str, ParsedSymbol, ColumnTableMetricTag) -> ParsedColumnMetricTag
-    parsed_column = _parse_symbol(mib, metric_tag['column'])
+    column = metric_tag['column']
+
+    if isinstance(column, dict) and 'table' in column:
+        # Prevent a common error due to bad indentation...
+        raise ConfigurationError(
```

Comment:
(nit) ConfigurationError should be pytest.fail ?

Reply:
This code is not in a test - I think we do want a `ConfigurationError` here to notify user that the profile is invalid.

Reply:
Misread 🤦